composer.json

```diff
@@ -50,6 +51,11 @@
     "extra": {
         "composer-exit-on-patch-failure": true,
         "enable-patching": true,
+        "patches": {
+            "openeuropa/oe_content": {
```

Comment:
we are using oe_content v2 but patching against v3, not good, this breaks at install for me even if the patch applies cleanly.
We should use the dev branch and epic once merged; 3.x once epic merged; and stable once released.